packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -1049,6 +1052,9 @@ export class PipelineEditor extends React.Component<
     }
 
     const runtimes = await PipelineService.getRuntimes();
```

Comment:
Given we're now adding a local runtime to this list we should update this so it will still work if no runtimes are configured
```suggestion
    const runtimes = await PipelineService.getRuntimes(false);
```